watertap/unit_models/nanofiltration_ZO.py

```diff
@@ -329,6 +379,9 @@ def initialize(
         init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="unit")
         solve_log = idaeslog.getSolveLogger(blk.name, outlvl, tag="unit")
         # Set solver options
+        if optarg is None:
+            optarg = {'bound_push': 1e-8}
```

Comment:
Is this needed here?

Reply:
Not necessarily- I was just trying to match the new option based on your scaling work. I think it may have worked without the bound push, so I'll double-check and revise if it does work without it.